crates/wdl-analysis/src/stdlib.rs

```diff
@@ -4116,10 +4379,14 @@ task select_all {
                         .min_version(SupportedVersion::V1(V1::One))
                         .type_parameter("K", PrimitiveTypeConstraint)
                         .any_type_parameter("V")
-                        .parameter(GenericMapType::new(
-                            GenericType::Parameter("K"),
-                            GenericType::Parameter("V")
-                        ))
+                        .parameter(
+                            "map",
+                            GenericMapType::new(
+                                GenericType::Parameter("K"),
+                                GenericType::Parameter("V"),
+                            ),
+                            "Map to convert to Pairs.",
```

Comment:
Is this going to be a markdown field? If so, it probably needs to be `Pair`s. Same with things like `Map`.